### PR TITLE
[Move] td-logger: Minor cleanups for td-logger crate

### PR DESCRIPTION
**Pull request by haowqs**
Signed-off-by: haowx <weix.hao.intel.com>

### CODE REVIEW

**Comment by jyao1** [comment]
Is that really clean up?

I feel the main is correct.

File: td-logger/src/logger.rs [hunk 1]
```diff
@@ -2,28 +2,39 @@
...
 pub const LOG_LEVEL_NONE: usize = 0;
 
 pub const LOG_MASK_COMMON: u64 = 0x1;
+// Core - Boot Service (BIT1 ~ BIT15)
+pub const LOG_MASK_PROTOCOL: u64 = 0x2;
+pub const LOG_MASK_MEMORY: u64 = 0x4;
+pub const LOG_MASK_EVENT: u64 = 0x8;
+pub const LOG_MASK_IMAGE: u64 = 0x10;
+// Core - Runtime Service (BIT16 ~ BIT 23)
+pub const LOG_MASK_VARIABLE: u64 = 0x10000;
```
Comment (jyao1) [inline td-logger/src/logger.rs:24]: Why we need add those?
I think main is correct.

**Comment by jyao1** [comment]
@jiangliu , I think we run into the case, that main branch fixed the issue, but staging branch does not.
Then people just gets confused on which one is correct. 

I think before we fully migrate to main, any fix in main must also be present in staging.

**Comment by jiangliu** [comment]
> @jiangliu , I think we run into the case, that main branch fixed the issue, but staging branch does not. Then people just gets confused on which one is correct.
> 
> I think before we fully migrate to main, any fix in main must also be present in staging. To eliminate the confusing, should be drop this patch, then create another one to merge the fix from trunk to staging?

I think we should do that, eventually we should merge the staging into the main branch to keep the development history.

**Review by jyao1 — CHANGES_REQUESTED** [review]
Would you please check which one the latest? Main or staging?
Now I feel main is right. If so, we need back port from main to staging.

**Pull request closed without merging**